api/src/main/java/org/apache/unomi/api/Profile.java

```diff
@@ -94,6 +95,30 @@ public Object getProperty(String name) {
         return properties.get(name);
     }
 
+    /**
+     * Retrieves the value of the nested property identified by the specified name.
+     *
+     * @param name the name of the property to be retrieved, splited in the nested properties with "."
+     * @return the value of the property identified by the specified name
+     */
+    public Object getNestedProperty(String name) {
+        if (!name.contains(".")) {
```

Comment:
I'm wondering if there is a way we could avoid this code duplication but I don't know where to put it unless we create some kind of static class in the API.

Reply:
Thank you for the comment. I'm not sure if I know where to put the common code, what do you think?

Reply:
Yeah let's leave it like this for the time being but we'll probably have to refactor at some point.